src/components/Square.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+const Square = ({
+  i,
+  squares,
+  setSquares,
+  remainingAlphabet,
+  setRemainingAlphabet,
+  errorMessage,
+  setErrorMessage,
+  maxNumberConsonants,
```

Comment:
Up to you, but React.context is built to avoid having to pass these props like setErrorMessage/maxNumberConsonants down multiple components. So you could add a context to App to pass them down

Reply:
Understood..... but I'm not comfortable enough yet to do this.